src/creature.h

```diff
@@ -185,6 +185,11 @@ class Creature
         dealt_projectile_attack projectile_attack( const projectile &proj, const tripoint &target,
                                                    double total_dispersion );
 
+        /**
+         * Return the chance that a projectile attack will hit with at least the given accuracy
```

Comment:
This is already a doxygen comment AFAIK and it appears in the generated documentation, what do you mean by "Function comments should be doxygen style".